README.md

```diff
@@ -1,37 +1,39 @@
 > WIP
 
-# Radix Monorepo
+# Radix
 
 This repository is a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md) powered by [Lerna](https://github.com/lerna/lerna) and [yarn workspaces](https://yarnpkg.com/lang/en/docs/cli/workspaces/).
 
-### Develop
+## Contributing
 
-#### Install dependencies
+There are many ways to contribute to the Radix and it's packages. Please first [create an issue](https://github.com/modulz/radix/issues/new) with your proposal, and from there we can start a conversation.
 
-```sh
-yarn
-```
+#### Getting Started
 
-#### Run website + Radix
+Bootstrap the repo by simply running:
 
-```sh
-yarn start
+```shell
+yarn             # Installs dependencies and links packages
+yarn start       # Develop Radix & it's Website
 ```
 
-### Build & Release
+**Other Packages**
 
-#### Build
+Check the `readme.md` of other packages, like `@modulz/radix-icons`, to get more details on contributing.
 
-```sh
-yarn build
-```
+#### Publishing
 
-#### Release
+We use `lerna publish` for managing the versions of packages in the monorepo. Since we also use [conventional-commits](https://www.conventionalcommits.org), changelogs will be generated automatically.
 
-```sh
-yarn release
+Once all commits have been made, from the root of the monorepo simply run:
+
+```shell
+yarn release         # prompts modified packages, generates changelogs and publishes to npm
+yarn release:github  # does same as above, but also generates a github release
 ```
 
+> _Please [follow these instructions](https://github.com/lerna/lerna/tree/master/commands/version#--github-release) by Lerna to authenticate for Github Releases._
```

Comment:
Also I think if we haven't had time to test it properly let's not use it yet.

I really wanna make sure anything we do in this repo is 100% clear and we know exactly why we're doing it (and do we really need it?)

Reply:
For sure - to this end you may want to consider, if you haven't already, encapsulating the version publishing & changelog generation inside of CI steps instead of on local machines.